CMakeLists.txt

```diff
@@ -1,7 +1,24 @@
 cmake_minimum_required(VERSION 2.8.12)
 project(cppzmq)
 
-find_package(ZeroMQ REQUIRED)
+find_package(ZeroMQ)
+
+# libzmq autotools install: fallback to pkg-config
+if(NOT ZeroMQ_FOUND)
+    find_package(PkgConfig)
+    pkg_check_modules(PC_LIBZMQ QUIET libzmq)
+
+    set(ZeroMQ_FOUND ON)
+    set(ZeroMQ_VERSION ${PC_LIBZMQ_VERSION})
+    find_library(ZeroMQ_LIBRARY NAMES libzmq.so
+                 PATHS ${PC_LIBZMQ_LIBDIR} ${PC_LIBZMQ_LIBRARY_DIRS})
+    find_library(ZeroMQ_STATIC_LIBRARY NAMES libzmq.a
+                 PATHS ${PC_LIBZMQ_LIBDIR} ${PC_LIBZMQ_LIBRARY_DIRS})
+endif()
+
+if(NOT ZeroMQ_FOUND)
+    message(FATAL_ERROR "ZeroMQ was NOT found!")
+endif()
 
 if (ZeroMQ_FOUND AND (NOT TARGET libzmq OR NOT TARGET libzmq-static))
```

Comment:
@herbrechtsmeier do you have an idea how I can define those targets again if `ZeroMQ_LIBRARY` and `ZeroMQ_STATIC_LIBRARY` are found (see above)